wpilibj/src/main/java/edu/wpi/first/wpilibj/experimental/drive/DifferentialDrive.java

```diff
@@ -107,6 +107,22 @@ public WheelSpeeds(double left, double right) {
       this.left = left;
       this.right = right;
     }
+
+
+    /**
+     * Normalizes the wheel speeds to a symmetric range with the given
+     * magnitude.
+     *
+     * @param maxSpeed Maximum wheel speed in normalization range
+     *                 [-maxSpeed..maxSpeed].
+     */
+    public void normalize(double maxSpeed) {
+      double maxMagnitude = Math.max(Math.abs(left), Math.abs(right));
+      if (maxMagnitude > maxSpeed) {
+        left /= maxMagnitude * maxSpeed;
```

Comment:
Why are other classes 
`out *= magnitude/speed`

but this is 
`out /= magnitude*speed`

Reply:
Dangit. I missed that one.

Reply:
Is one better than the other?

Reply:
Yeah. One is actually correct. :)